packages/trader/src/App/Components/Form/CompositeCalendar/composite-calendar-mobile.jsx

```diff
@@ -112,7 +112,23 @@ const CompositeCalendarMobile = React.memo(
             }
 
             if (key === 'to') {
-                setTo(e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '');
+                // Handling future dates input for IOS devices
+                const value = e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '';
+
+                setTo(value);
```

Comment:
Is it possible to move this call inside the below condition as well to avoid extra call? I think it calls this function 3 times in some cases